src/web/modules/gateway_accounts/detail.njk

```diff
@@ -186,7 +193,32 @@
       })
       }}
     {% endif %}
+    {% if account.allow_moto === true %}
+      <form method="POST" action="/gateway_accounts/{{ gatewayAccountId }}/moto_payments">
+        {{ govukButton({
+          text: "Disable MOTO payments"
+          })
+        }}
+        <input type="hidden" name="_csrf" value="{{ csrf }}">
+      </form>
+    {% endif %}
+  </div>
```

Comment:
I think it might be clearer on the page if we maintain a different header and section for moto payments when they are enabled. The header could be something like "MOTO payments are enabled", and we could possibly maintain the second paragraph of the blurb to make it clear to a toolbox user what this means for the account.